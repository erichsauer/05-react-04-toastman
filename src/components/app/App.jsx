import React from 'react';
import Toastman from '../../containers/Toastman';

export default function App() {
  return (
    <>
      <Toastman />
    </>
  );
}
