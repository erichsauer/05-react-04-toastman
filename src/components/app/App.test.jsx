import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './App';

describe('App component', () => {
  afterEach(() => cleanup());
  it('makes the CI pass', () => {});
});
