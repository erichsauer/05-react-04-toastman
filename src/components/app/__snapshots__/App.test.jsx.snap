// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toastman component should be able to make a get/post/put/patch/delete request to a user-specified url and send along user-entered json 1`] = `
<span
  class="__json-string__"
  style="color:#7fd6fa;"
>
  "You say that those brains are making everyone on Earth stupid. Oh... stupider."
</span>
`;
